Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:dubnium
```

Comment:
Will an Apline image suffice ?

Reply:
I think that an alpine image might work. I may have to assess though if it is necessary to still maintain some packages in the image. But yeah alpine is a great idea.

Reply:
Alright cool. If possible lets use an alpine image throughout.